doc/concepts/cable_cell.rst

```diff
@@ -139,7 +159,7 @@ morphology. Each fork point in the morphology will be the responsibility of
 a single CV, and as a special case a zero-volume CV can be used to represent
 a single fork point in isolation.
 
-.. _cable-density-mechs:
+.. _cablecell-density-mechs:
```

Comment:
The subsection on Discretisation here is out of place now, but it's also made redundant given that the longer discussion has been brought into this file from the C++ API. Can you remove these lines 150–160 and redirect any references to `cablecell-discretization` to `cablecell-cv-policies` instead?